index.d.ts

```diff
@@ -76,6 +76,7 @@
      * Base URI Defaults to `https://media.graphcms.com`
      */
     baseURI?: string
+    loading?: string
```

Comment:
i think we can improve the type for this by using `HTMLImageElement['loading']`

Reply:
good call, that's much better